force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +48,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        if (age >= 18) {
+            return true;
+        }
+
+        return false; // Replace null with the variable you used to store the result
```

Comment:
good work. Can you solve this with one line. Think about what a < or > evaluate to.